db/migrate/20150818181427_update_tenant_divisible_on_existing_rows.rb

```diff
@@ -1,7 +1,8 @@
 class UpdateTenantDivisibleOnExistingRows < ActiveRecord::Migration
   class Tenant < ActiveRecord::Base
   end
```

Comment:
You want to add code just incase we'll add STI to every model in migrations?
I'm leaning away from this